src/OrleansRuntime/Placement/IPlacementContext.cs

```diff
@@ -68,9 +72,9 @@ public static string GetGrainTypeName(this IPlacementContext @this, GrainId grai
             return @this.GetGrainTypeName(grainId.GetTypeCode(), genericArguments);
         }
 
-        public static void GetGrainTypeInfo(this IPlacementContext @this, GrainId grainId, out string grainClass, out PlacementStrategy placement, string genericArguments = null)
+        public static void GetGrainTypeInfo(this IPlacementContext @this, GrainId grainId, out string grainClass, out PlacementStrategy placement, out MultiClusterRegistrationStrategy activationStrategy, string genericArguments = null)
```

Comment:
I agree. I started doing this but it looks like it touches a fair number of files so it is probably better to do it after we merge this PR, like you say.